Utils/Dataflow/pyDKB/common/hdfs.py

```diff
@@ -22,6 +23,16 @@ def check_stderr(proc, timeout=None, max_lines=1):
     if MAX_LINES == None, output all the STDERR.
 
     Return value is the subprocess` return code.
+    
+    :param proc: subrocess
+    :type proc: subprocess
+    :param timeout: time limit for operation
+    :type timeout: number (int)
+    :param max_lines: maximum quantity of lines
+    :type max_lines: number (int)
+    
+    :return: subprocess return code, set by poll()
```

Comment:
The return code is not *set* by `poll()` -- it is returned by the subprocess, and can be *accessed* by `poll()`.
I believe it will be Ok to simply say that it is a "subprocess return code".